tiledb/sm/cpp_api/version.h

```diff
@@ -36,60 +36,18 @@
 #include "tiledb.h"
 
 #include <iostream>
+#include <tuple>
 
-namespace tiledb {
-
-/** TileDB version. Format: `major_minor_rev`. */
-class TILEDB_EXPORT Version {
- public:
-  /* ********************************* */
-  /*                API                */
-  /* ********************************* */
-
-  /** Returns the major number. */
-  inline int major() const {
-    return major_;
-  }
-
-  /** Returns the minor number. */
-  inline int minor() const {
-    return minor_;
-  }
-
-  /** Returns the patch number. */
-  inline int patch() const {
-    return patch_;
-  }
+#undef major
```

Comment:
Do we need these undefs now that we're returning a tuple?

Reply:
Good catch